### ISSUE TITLE
[BUG] Screenshots are displayed in low resolution on the store page

### ISSUE THREAD

**Issue by joosniz** [IsThereAnyDeal/AugmentedSteam#1724]
### Expected Behavior
Screenshot resolution should be equal to or greater than the dimensions that they are displayed at, just as is the case for videos, at least when using the expanded view mode that makes videos and screenshots bigger. This is especially true considering that screenshots are all available in 1920x1080, and can be viewed by editing the resolution in the image source links.

### Actual Behavior
Screenshots are presented with the resolution 600x338, but upscaled drastically depending on various factors (measured at 2114x1190 on my laptop, covering most of the 13" screen and being only barely smaller than the full-screen version). 

### Steps to Reproduce the Problem

  1. View any store page for a game
  2. Compare the difference in clarity between screenshots and videos
  3. Press the expand button when viewing a screenshot
  4. Press the screenshot to view it in fullscreen
  5. Once again pay attention to the difference in clarity

### Specifications

  - Augmented Steam Version: Version 2.5.0
  - Browser Version: Microsoft Edge version 117.0.2007.0

**Comment by candela97** [comment]
Yeah I think we should use higher resolution screenshots and videos for the expanded mode.

**Comment by candela97** [comment]
Doable for videos (I removed it a while back), but for screenshots, Steam sets the `src` on load so it'll always switch to the 600x338 resolution:
```js
HighlightPlayer.prototype.LoadScreenshot = function( id )
{
	var $Target = $JFromIDOrElement( 'highlight_screenshot_' + id );
	if ( $Target.length )
	{
		var url = this.GetScreenshotURL( id, '600x338' );
		var $Img = $Target.find('img');
		if ( $Img.attr( 'src' ) != url )
			$Img.attr( 'src', url );

	}
 }
```
The only workaround I came up with is inserting a fake element...
